components/general/OurClients.tsx

```diff
@@ -0,0 +1,58 @@
+import styles from "@/styles/OurClients.module.css";
+import { fontSatoshi, fontSyne } from "@/pages/_app";
+import IconsList from "./IconList";
+
+const OurClients = () => {
+  const iconsList = [
+    {
+      id: 1,
+      aria_label: "icon1",
+      icon: "/icons/icon1.svg",
+      alt: "icon1",
+    },
+    {
+      id: 2,
+      aria_label: "icon2",
+      icon: "/icons/icon2.svg",
+      alt: "icon2",
+    },
+    {
+      id: 3,
+      aria_label: "icon3",
+      icon: "/icons/icon3.svg",
+      alt: "icon3",
+    },
+    {
+      id: 4,
+      aria_label: "icon4",
+      icon: "/icons/icon4.svg",
+      alt: "icon4",
+    },
+    {
+      id: 5,
+      aria_label: "icon5",
+      icon: "/icons/icon5.svg",
+      alt: "icon5",
+    },
+    {
+      id: 6,
+      aria_label: "icon6",
```

Comment:
Area-label is the one used for accessibility, You don't want your mobile to read these icons as "icon 1, icon 2". You want your mobile to read these as "Super food icon, Create studio icon".